remote_kernel_provider/lifecycle_manager.py

```diff
@@ -439,7 +439,7 @@ def _enforce_authorization(self, **kwargs):
             self._raise_authorization_error(self.kernel_manager.kernel_username, "not authorized")
 
         # If authorized users are non-empty, ensure user is in that set.
-        if self.authorized_users.__len__() > 0:
+        if len(self.authorized_users) > 0:
```

Comment:
Isn't the deepCode stuff saying this should be `if self.authorized_users:` instead?

https://www.deepcode.ai/app/gh/gateway-experiments/remote_kernel_provider/855875a8259ba84ba5cdc9dba8fa2eeb29d19b12/gateway-experiments/remote_kernel_provider/7383c4cc05d8769d46ffa027f039d028ae4c3a3d/_/%2Fremote_kernel_provider%2Flifecycle_manager.py/python%2Fpylint%2FC1801/code/

Same goes for https://github.com/gateway-experiments/remote_kernel_provider/pull/25/files#diff-7ce94dc9a978ed8370e0872d68968dc2R180

Reply:
Indeed, the original issue raised was related to __len__, but now the PR review has a different suggestion. Good catch.